controllers/applicationvolumereplication_controller.go

```diff
@@ -105,8 +105,7 @@ func (r *ApplicationVolumeReplicationReconciler) Reconcile(ctx context.Context,
 		return ctrl.Result{}, errorswrapper.Wrap(err, "failed to list subscriptions")
 	}
 
-	placementDecisions := r.processSubscriptionList(subscriptionList)
-
+	placementDecisions, requeue := r.processSubscriptionList(subscriptionList)
```

Comment:
This has been already corrected in the pause/unpause PR.  So you might hold-on to this so that it doesn't cause a merge nightmares.